test/test-html.js

```diff
@@ -0,0 +1,273 @@
+
+/** Elements that are allowed in all properties. */
+const ALLOWED_GLOBAL_ELEMENTS = [
+  // Force newline
+  'code',
+  'em',
+  'kbd',
+  'strong',
+];
+
+/**
+ * Elements allowed only in specific properties.
+ *
+ * @type {{[property: string]: string[]}}
+ */
+const ALLOWED_PROPERTY_ELEMENTS = {
+  notes: ['a'],
+};
+
+/**
+ * Special attributes that are limited only to specific elements.
+ *
+ * @type {{[element: string]: string[]}}
+ */
+const ALLOWED_ELEMENT_ATTRIBUTES = {
+  a: ['href'],
+};
+
+/**
+ * Calls a defined callback function on each element of an array, and returns an array that contains the results.
+ * @template T, U
+ * @param {T[]} array The array to process.
+ * @param {(value: T, index: number, array: T[]) => U | U[]} callbackfn A function that accepts up to three arguments. The flatMap method calls the callbackfn function one time for each element in the array.
+ * @param {unknown} [thisArg] An object to which the this keyword can refer in the callbackfn function. If thisArg is omitted, undefined is used as the this value.
+ * @return {U[]}
+ */
+function flatMapArray(array, callbackfn, thisArg = this) {
+  /** @type {U[]} */
+  let result = [];
+  for (let i = 0; i < array.length; i++) {
+    result.concat(callbackfn.call(thisArg, array[i], i, array));
+  }
+  return result;
+}
+
+/**
+ * Checks all properties that may contain HTML in the `CompatStatement`.
+ *
+ * @param {CompatStatement} compat The browser compatibility statement.
+ * @param {string} path The path of the feature.
+ * @param {Logger} logger The logger.
+ */
+function checkCompatStatement(compat, path, logger) {
+  let hasErrors = false;
+  if (compat.description) {
+    lintHTMLString(
+      compat.description,
+      'description',
+      `${path}.__compat.description`,
+      logger,
+      // Allow `<a>` when `mdn_url` is not specified:
+      compat.mdn_url ? undefined : ['a'],
+    );
+  }
+
+  flatMapArray(
+    flatMapArray(
+      Object.keys(compat.support).map(b => compat.support[b]),
+      v => v,
+    ).filter(o => o.notes),
+    v => v.notes,
+  ).forEach(note => lintHTMLString(note, 'notes', path, logger));
+
+  for (const browser in compat.support) {
+    /** @type {SimpleSupportStatement[]} */
+    const supportStatements = [];
+    if (Array.isArray(compat.support[browser])) {
+      Array.prototype.push.apply(supportStatements, compat.support[browser]);
+    } else {
+      supportStatements.push(/** @type {any} */ (compat.support[browser]));
+    }
+
+    for (const statement of supportStatements) {
+      const notes = Array.isArray(statement.notes)
+        ? statement.notes
+        : [statement.notes];
+
+      for (const note of notes) {
+        lintHTMLString(
+          note,
+          'notes',
+          `${path}.__compat.support.${browser}..notes`,
```

Comment:
Same thing as up above.  Since `notes` is already specified and `__compat` is implied, we can simplify the line:

```suggestion
          `${path} (${browser})`,
```

Note that this is also due to text formatting consistency.  All of the other test files use the shorter path (example `api.AbortController has the following browsers...`).  The browser name in parenthesis is totally a styling preference we have relatively free reign over here, though!